packages/css-engine/src/core/rules.ts

```diff
@@ -142,9 +142,19 @@ export class FontFaceRule {
     this.options = options;
   }
   get cssText() {
+    const decls = [];
     const { fontFamily, fontStyle, fontWeight, fontDisplay, src } =
       this.options;
-    return `@font-face {\n  font-family: ${fontFamily}; font-style: ${fontStyle}; font-weight: ${fontWeight}; font-display: ${fontDisplay}; src: ${src};\n}`;
+    decls.push(
+      `font-family: ${
+        /\s/.test(String(fontFamily)) ? `"${fontFamily}"` : fontFamily
```

Comment:
Why cast to string? Type is not sufficient?

Reply:
true, not necessary after rewrite